app/services/food_service.rb

```diff
@@ -0,0 +1,13 @@
+class FoodService
+  def self.conn
+    conection = Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |faraday|
+      faraday.params['api_key'] = ENV['fdc_api_key']
+    end
+  end
+
+  def self.food_search(food_name)
+    response = conn.get("foods/search?query=#{food_name}")
```

Comment:
you could also add here `pageSize=10` which will narrow the results to 10 at the call. 
```
response = conn.get("foods/search?query=#{food_name}&pageSize=10")
```